### PR TITLE
Fixed some gui issues when conversion/validation returns nulls in fields.

### PR DESCRIPTION
**Pull request by cday-semanticbits**
Added null protection on fields so the report will still show if they are null.

### CODE REVIEW

File: src/main/java/mat/server/service/FhirValidationReportService.java [hunk 1]
```diff
@@ -169,9 +174,15 @@ private String generateValidationReport(ConversionResultDto conversionResultDto,
 
             }
 
-            Instant instant = Instant.parse(conversionResultDto.getModified());
-            paramsMap.put("runDate", DateUtility.formatInstant(instant, DATE_FORMAT));
-            paramsMap.put("runTime", DateUtility.formatInstant(instant, TIME_FORMAT));
+            if (conversionResultDto.getModified() != null) {
+                Instant instant = Instant.parse(conversionResultDto.getModified());
```
Comment (serhii-ilin) [inline src/main/java/mat/server/service/FhirValidationReportService.java]: JFYI. You could do that from the template with a default value.

File: src/main/java/mat/server/ExportServlet.java [hunk 2]
```diff
@@ -554,6 +561,18 @@ private void exportSimpleXML(HttpServletResponse resp, String id, String type, M
         }
     }
 
+    private void exportMatXML(HttpServletResponse resp, String id, String type, Measure measure) throws Exception {
+        ExportResult export = getService().getMatXml(id);
+        if (SAVE.equals(type)) {
```
Comment (serhii-ilin) [inline src/main/java/mat/server/ExportServlet.java]: Is this part of a different JIRA? How is this related to the conversion?

File: src/main/java/mat/server/service/FhirValidationReportService.java [hunk 3]
```diff
@@ -104,14 +105,15 @@ private String generateValidationReport(ConversionResultDto conversionResultDto,
...
+                if(StringUtils.isNotBlank(conversionResultDto.getErrorReason())) {
                     paramsMap.put("ErrorReason", conversionResultDto.getErrorReason());
                 }
                 // ValueSet FHIR validation errors
                 List<FhirValidationResult> valueSetFhirValidationErrors = new ArrayList<>();
                 if (CollectionUtils.isNotEmpty(conversionResultDto.getValueSetConversionResults())) {
                     for (ValueSetConversionResults valueSetConversionResults : conversionResultDto.getValueSetConversionResults()) {
-                        if (Boolean.FALSE.equals(valueSetConversionResults.getSuccess())) {
+                        if (valueSetConversionResults.getSuccess() != null &&
+                                false == valueSetConversionResults.getSuccess()) {
```
Comment (jkotanchik-SB) [inline src/main/java/mat/server/service/FhirValidationReportService.java]: Why can't java just figured this out for itself.
